packages/snap-controller/src/Search/SearchController.ts

```diff
@@ -744,12 +785,14 @@ function getSearchSchemaData({
 		q: params.search?.query?.string || '',
 		correctedQuery: store.search?.originalQuery?.string ? store.search?.query?.string : undefined,
 		...filters,
-		sort: [
-			{
-				field: store.sorting.current?.field,
-				dir: store.sorting.current?.direction as AutocompleteSchemaDataSortInnerDirEnum,
-			},
-		],
+		sort: store.sorting.options
+			.filter((sort) => sort.active)
+			?.map((sort) => {
+				return {
+					field: sort.field,
+					dir: sort.direction as AutocompleteSchemaDataSortInnerDirEnum,
+				};
+			}),
```

Comment:
As mentioned in AutocompleteController comment, we will likely need to use the request parameters instead.